### PR TITLE
Clean up recipe

### PR DESCRIPTION
**Pull request by dionhaefner**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by PhilipVinc** [comment]
You should un comment the includes for test and add back the run_test.sh I removed a few commits ago because it was broken, but now should work.

**Comment by dionhaefner** [comment]
Like this?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,23 +32,20 @@ requirements:
     - python
     - numpy
     - jax >=0.2
+    - jaxlib
```
Comment (PhilipVinc) [inline recipe/meta.yaml:35]: Why?
Jax on conda already depends on jaxlib

**Comment by dionhaefner** [comment]
There was a warning about it here:

https://github.com/conda-forge/mpi4jax-feedstock/pull/20

But if you have strong feelings about it I can remove it.

**Comment by PhilipVinc** [comment]
No that's fine

**Pull request merged**